sw/device/silicon_creator/lib/drivers/rnd_unittest.cc

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/drivers/rnd.h"
+
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/csr.h"
+#include "sw/device/lib/base/testing/mock_abs_mmio.h"
+#include "sw/device/silicon_creator/lib/base/mock_csr.h"
+#include "sw/device/silicon_creator/lib/base/mock_sec_mmio.h"
+#include "sw/device/silicon_creator/lib/drivers/mock_otp.h"
+#include "sw/device/silicon_creator/testing/mask_rom_test.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "otp_ctrl_regs.h"
+#include "rv_core_ibex_regs.h"
+
+namespace rng_unittest {
+namespace {
+using ::testing::Return;
+
+class RngTest : public mask_rom_test::MaskRomTest {
+ protected:
+  const uint32_t rng_addr_ = TOP_EARLGREY_RV_CORE_IBEX_CFG_BASE_ADDR;
```

Comment:
This should be `constexpr`.

Reply:
Using `constexpr` would require it to be static and also initialized outside of the class.
So I rather decided to use a enum.